unified_targets/configs/AIRBUSF405.config

```diff
@@ -0,0 +1,133 @@
+# Betaflight / STM32F405 (S405) 4.0.0 Apr  3 2019 / 14:30:44 (22b9f3453) MSP API: 1.41
+
+board_name AIRBUSF405
+manufacturer_id HGLR
+
+# resources
+resource BEEPER 1 B04
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 A03
+resource MOTOR 4 A02
+resource MOTOR 5 A01
+resource MOTOR 6 A08
+resource PPM 1 B14
+resource PWM 1 B14
+resource PWM 2 B15
+resource PWM 3 C06
+resource PWM 4 C07
+resource PWM 5 C08
+resource PWM 6 C09
+resource SONAR_TRIGGER 1 A01
+resource SONAR_ECHO 1 A08
+resource LED_STRIP 1 A01
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 6 C07
+resource INVERTER 1 C00
+resource LED 1 B05
+resource SPI_SCK 1 A05
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 3 C12
+resource ESCSERIAL 1 B14
+resource ADC_BATT 1 C02
+resource ADC_CURR 1 C01
+resource FLASH_CS 1 B03
+resource OSD_CS 1 A15
+resource RX_SPI_CS 1 D02
+resource RX_SPI_EXTI 1 A00
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C05
+
+# timer
+timer B14 2
+timer B15 2
+timer C06 1
+timer C07 1
+timer C08 1
+timer C09 1
+timer B00 1
+timer B01 1
+timer A03 0
+timer A02 0
+timer A01 1
+timer A08 0
+timer A09 0
+timer A10 0
+
+# dma
+dma ADC 2 1
+# ADC 2: DMA2 Stream 3 Channel 1
+dma pin C06 0
+# pin C06: DMA2 Stream 2 Channel 0
+dma pin C07 0
+# pin C07: DMA2 Stream 2 Channel 0
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin A03 1
+# pin A03: DMA1 Stream 6 Channel 3
+dma pin A02 0
+# pin A02: DMA1 Stream 1 Channel 3
+dma pin A01 0
+# pin A01: DMA1 Stream 4 Channel 6
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+
+# master
+set gyro_to_use = FIRST
+set align_mag = DEFAULT
```

Comment:
Please remove all of the default settings from this section by following the instructions in https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md.

Reply:
Thank you very much for reminding.